eo-compiler/src/main/java/org/eolang/compiler/syntax/Attribute.java

```diff
@@ -54,15 +54,41 @@ public Attribute(final String type, final String name) {
     }
 
     /**
-     * Generate java code for object attribute.
+     * Generate java field code for object attribute.
      *
-     * @return Generated java code.
+     * @return Generated java field code.
      */
-    public String java() {
+    public String asField() {
```

Comment:
Instead of three methods: `asField`, `asCtorParam` and `asCtorInitializer`, I'd leave one:
`String java(Code code) { return code.java(this.type, this.name); }` and introduce three implementations of `Code` interface: `Field`, `CtorParam` and `CtorInitializer`.

The body of `Field::java` would be: `return String.format("private final %s %s;", type, name);`

Of course there could be a better name instead of `Code` ;)
What do you think?